packages/cli/src/commandsConfig.ts

```diff
@@ -379,6 +379,10 @@ const commandsConfig = {
         flags: '-p --preset <preset>',
         description: '(DEPRECATED) Preset of the deployment to alter',
       },
+      {
+        flags: '--no-publish',
+        description: 'Do not publish the result to the regisgtry, just pin the new IPFS artifacts.',
```

Comment:
it doesn't normally publish "to the registry", it pubishes to the local tags which can then be published

Reply:
Ah perfect, thought that it was to the registry, will remove it then.